cfg/cis-1.9/policies.yaml

```diff
@@ -0,0 +1,398 @@
+---
+controls:
+version: "cis-1.9"
+id: 5
+text: "Kubernetes Policies"
+type: "policies"
+groups:
+  - id: 5.1
+    text: "RBAC and Service Accounts"
+    checks:
+      - id: 5.1.1
+        text: "Ensure that the cluster-admin role is only used where required (Automated)"
+        audit: |
+          kubectl get clusterrolebindings -o=custom-columns=NAME:.metadata.name,ROLE:.roleRef.name,SUBJECT:.subjects[*].name --no-headers | while read -r role_name role_binding subject
+          do
+            if [[ "${role_name}" != "cluster-admin" && "${role_binding}" == "cluster-admin" ]]; then
+              is_compliant="false"
+            else
+              is_compliant="true"
+            fi;
+            echo "**role_name: ${role_name} role_binding: ${rolebinding} subject: ${subject} is_compliant: ${is_compliant}"
+          done
+        use_multiple_values: true
+        tests:
+          test_items:
+            - flag: "is_compliant"
+              compare:
+                op: eq
+                value: true
+        remediation: |
+          Identify all clusterrolebindings to the cluster-admin role. Check if they are used and
+          if they need this role or if they could use a role with fewer privileges.
+          Where possible, first bind users to a lower privileged role and then remove the
+          clusterrolebinding to the cluster-admin role : kubectl delete clusterrolebinding [name]
+          Condition: is_compliant is false if rolename is not cluster-admin and rolebinding is cluster-admin.
+        scored: true
+      - id: 5.1.2
+        text: "Minimize access to secrets (Automated)"
+        audit: "echo \"canGetListWatchSecretsAsSystemAuthenticated: $(kubectl auth can-i get,list,watch secrets --all-namespaces --as=system:authenticated)\""
+        tests:
+          test_items:
+            - flag: "canGetListWatchSecretsAsSystemAuthenticated"
+              compare:
+                op: eq
+                value: no
+        remediation: |
+          Where possible, remove get, list and watch access to Secret objects in the cluster.
+        scored: true
+      - id: 5.1.3.1
```

Comment:
```suggestion
      - id: 5.1.3
```

Reply:
As mentioned in the PR description, I chose to split 5.1.3 into two other checks (5.1.3.1 and 5.1.3.2) that are the referenced artifacts in CIS Workbench. This gives more accuracy and reduce the complexity of having to test 5.1.3.1 and 5.1.3.2 within the same check.
To make sure we keep the same references, I've added a `Parent: 5.1.3` in the remediation.

Here are the details:

<img width="618" alt="image" src="https://github.com/aquasecurity/kube-bench/assets/12878731/44d51ca9-af75-477b-8798-080a99318355">

<img width="817" alt="image" src="https://github.com/aquasecurity/kube-bench/assets/12878731/c2d2658e-dd51-4aef-b652-b1b42eca0deb">

Reply:
IMHO, we should follow the CIS benchmark, don't split the remediation in this version. As you can see, both 5.1.3.1 and 5.1.3.2 are draft.

* This may break downstream.
* We may have to explain it to everyone who has questions about this.

@chen-keinan WDYT?

Reply:
I agree , CIS benchmark should be the exact guide to follow to avoid breaking changes

Reply:
@mozillazg I've just made the required changes for 5.1.3 and 5.1.6, see the commit https://github.com/aquasecurity/kube-bench/pull/1617/commits/85cecda50fe6f16009cdd70b1f41370c17394d77.
I've also modified the PR description to reflect these changes, with the tests.